src/LaTeXStrings.jl

```diff
@@ -50,8 +50,8 @@ macro L_str(s, flags...) latexstring(s) end
 macro L_mstr(s, flags...) latexstring(s) end
 
 Base.write(io::IO, s::LaTeXString) = write(io, s.s)
-Base.show(io::IO, ::MIME"application/x-latex", s::LaTeXString) = write(io, s)
-Base.show(io::IO, ::MIME"text/latex", s::LaTeXString) = write(io, s)
+Base.show(io::IO, ::MIME"application/x-latex", s::LaTeXString) = (write(io, s); nothing)
+Base.show(io::IO, ::MIME"text/latex", s::LaTeXString) = (write(io, s); nothing)
```

Comment:
The correct fix is probably to call `print(io, s.s)`.

`print` is preferred for writing text because it is potentially encoding-aware, whereas `write(io, s.s)` will always write raw bytes.